doc/api/debugger.md

```diff
@@ -202,3 +202,23 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+The set of categories for which traces are recorded can be specified using the
+`--enabled-categories` flag followed by a list of comma separated category names.
+By default the `node` and `v8` categories are enabled.
+
+```txt
+node --enable-tracing --enabled-categories=v8,node server.js
+```
+
+Running Node.js with tracing enabled will produce log files that can be opened
+in the [`chrome://tracing`](https://www.chromium.org/developers/how-tos/trace-event-profiling-tool)
+tab of Chrome.
```

Comment:
The format is currently defined by the implementation in the `JSONTraceWriter` class. The original thought was that other output formats could be supported by providing alternate implementations of `node_trace_writer.cc` to support human readable or any other desired output formats. We could also explore translating json trace output but the workflow would probably be more smooth if the output format was controlled by a flag.